Mesh_3/include/CGAL/Mesh_3/Mesher_3.h

```diff
@@ -719,6 +720,20 @@ initialize()
   else
 #endif // CGAL_LINKED_WITH_TBB
   {
+
+    if (r_c3t3_.number_of_facets() == 0)
+    {
+      CGAL::warning_fail("c3t3.number_of_facets() == 0",
+        "CGAL/Mesh_3/Mesher_3.h",
+        725,
+        "Warning : The mesh refinement process can't start.\n"
+        "When calling refine_mesh_3(), the input `c3t3` should have been initialized and have "
+        "at least one facet in the complex. Try to solve this issue using :\n"
+        "\t- The automatic initialization provided by make_mesh_3()\n"
+        "\t- Adding more and better chosen points on the input surface\n"
+        "\t- defining the compilation macro CGAL_SEQUENTIAL_MESH_3_ADD_OUTSIDE_POINTS_ON_A_FAR_SPHERE\n");
```

Comment:
Are you sure `CGAL_SEQUENTIAL_MESH_3_ADD_OUTSIDE_POINTS_ON_A_FAR_SPHERE` will solve anything?

Reply:
I'm not sure, but it's a hint...
Do you think it's there only for benchmarking/performance reasons?

Reply:
We have added `CGAL_SEQUENTIAL_MESH_3_ADD_OUTSIDE_POINTS_ON_A_FAR_SPHERE` for robustness issue, to avoid to have facets of the c3t3 on the convex hull. I do not know if that can help the initialization of the triangulation.

Reply:
ok I'll remove it